### DISCUSSION TITLE
Add AstarGrid2D to 2D Navigation Overview?

### DISCUSSION THREAD

**Discussion by shak2** [godotengine/godot-proposals#10316] [General Discussions]
This page:

[https://docs.godotengine.org/en/stable/tutorials/navigation/navigation_introduction_2d.html](https://docs.godotengine.org/en/stable/tutorials/navigation/navigation_introduction_2d.html)

Does not contain a headline with detail of this important (new?) node:

[https://docs.godotengine.org/en/stable/classes/class_astargrid2d.html](https://docs.godotengine.org/en/stable/classes/class_astargrid2d.html)

I am new to pathfinding, so maybe there are other new 2D and 3D navigation nodes that are missing in the 2D and 3D pathfinding/navigation docs pages.

Please check and update as this is important information for completeness of these pages.

**Comment by AThousandShips** [comment]
This should be discussed as an issue [here](https://github.com/godotengine/godot-docs/issues)
